edge/pkg/edged/cadvisor/cadvisor_linux.go

```diff
@@ -0,0 +1,97 @@
+// +build cgo,linux
```

Comment:
This is not actually depending on linux, maybe remove _linux in filename, and remove corresponding build arg

Reply:
Done